tools/acle_template.tex

```diff
@@ -24,16 +23,19 @@
 \usepackage[defaultsans,scale=0.95]{lato}
 \usepackage{inconsolata}
 \renewcommand*\familydefault{\sfdefault}
-
-$if(geometry)$
-\usepackage[$for(geometry)$$geometry$$sep$,$endfor$]{geometry}
-$endif$
-
+\usepackage{graphicx}
+\usepackage[$if(landscape)$landscape,$endif$top=3.7cm,bottom=2.7cm,left=3cm,right=2cm,headsep=1.5cm,footskip=.5cm]{geometry}
 \usepackage{lastpage}
+\usepackage{dblfloatfix}
+\usepackage{subcaption}
```

Comment:
What are these for? For the titlepage? Please move them right before the redefinition of `\maketitle` if so.

Reply:
They're used for the grid image, yes.